diff --git a/App/container-api.ts b/App/container-api.ts
index 18d8f7978dd5f9dd7bde0790c1f584735de2782d..2f3964bb09dcdb195ed798a6f7c0695bb2cf51d0 100644
--- a/App/container-api.ts
+++ b/App/container-api.ts
@@ -22,7 +22,7 @@ import { AuthenticateStackParams, ScreenOptionsType } from './types/navigators'
 import { CustomNotification, CustomNotificationRecord } from './types/notification'
 import { Config } from './types/config'
 import { NotificationListItemProps } from './components/listItems/NotificationListItem'
-import { PINCreateHeaderProps } from 'components/misc/PINCreateHeader'
+import { PINCreateHeaderProps } from './components/misc/PINCreateHeader'
 
 export type FN_ONBOARDING_DONE = (
   dispatch: React.Dispatch<ReducerAction<unknown>>,
diff --git a/App/types/config.ts b/App/types/config.ts
index ad1a7351b07e5ef30200c5b322dade69705bf360..eda623050eff3d638199a9c673bc6dee463f82eb 100644
--- a/App/types/config.ts
+++ b/App/types/config.ts
@@ -1,4 +1,4 @@
-import { Locales } from "localization";
+import { Locales } from "../localization";
 import { PINSecurityParams } from "./security";
 import { SettingSection } from "./settings";
 import { Agent } from "@credo-ts/core";
